src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jv-jdbc-intro";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", getPassword());
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't load the JDBC driver.", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(DB_URL, DB_PROPERTIES);
+    }
+
+    private static String getPassword() {
+        String path = "src/main/resources/pswrd.txt";
+        try {
+            return Files.readString(Path.of(path));
```

Comment:
If you decided to read password from file, you should close resource or use BufferedReader in try-with-resources